packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -67,12 +67,7 @@ export interface ArticleMetadataPanelProps {
   onChange?(value: ArticleMetadata): void
 }
 
-export function ArticleMetadataPanel({
```

Comment:
@bishoymelek i don't think this file should be changed.

Reply:
@roosnic1 It has been removed a few days ago, can you double check?